sympy/stats/joint_rv_types.py

```diff
@@ -712,8 +719,10 @@ def GeneralizedMultivariateLogGamma(syms, delta, v, lamda, mu):
     ====
 
     If the GeneralizedMultivariateLogGamma is too long to type use,
-    ``from sympy.stats.joint_rv_types import GeneralizedMultivariateLogGamma as GMVLG``
-    If you want to pass the matrix omega instead of the constant delta, then use,
+
+    >>> from sympy.stats.joint_rv_types import GeneralizedMultivariateLogGamma as GMVLG
+
+    If you want to pass the matrix omega instead of the constant delta, then use
```

Comment:
This was discussed when I updated parts of the documentation and there were opinions that it maybe was not really relevant. However, now that you have made it a doctest, one should possibly go ahead and also use GMVLG to define a variable.